### ISSUE TITLE
getselectedrow() returning "attempt to call a nil value (method 'getselectedrow')"

### ISSUE THREAD

**Issue by gustavohmsilva** [yue/yue#75]
Hi guys;

Since I applied the temporary fix ```gui.Table = gui.TableModel``` I've been playing with the lib in Lua 5.3 and noticed that getselectedrow() on any table view return's me the following error:

```
lua5.3: teste2.lua:99: attempt to call a nil value (method 'getselectedrow')
stack traceback:
        teste2.lua:99: in main chunk
        [C]: in ?
```

I'm not sure if it is a new bug of a collateral effect of bug #74 , so I decided to share it either way.

**Comment by zcbenz** [comment]
Yeah it is a new bug, the `selectrow` and `getselectedrow` methods were not exported to language bindings.

**Comment by gustavohmsilva** [comment]
Ok then. Since I'm trying doing some applications with it, I'll keep sharing the problems I find here. Hope it helps.

**Comment by zcbenz** [comment]
Fixed in 0.6.3.
